src/index.js

```diff
@@ -103,6 +107,7 @@ export default function babel ( options ) {
 		intro () {
 			const helpers = Object.keys( bundledHelpers );
 			if ( !helpers.length ) return '';
+			if ( externalHelpers ) return '';
```

Comment:
This could be moved to the top of the method, as we can return immediately if we're using external helpers.